src/algo/DoublyLinkedList.js

```diff
@@ -388,6 +394,13 @@ export default class DoublyLinkedList extends Algorithm {
 	reset() {
 		this.size = 0;
 		this.nextIndex = this.resetIndex;
+		this.infoLabelID = this.nextIndex++;
```

Comment:
```suggestion
```

Reply:
Same as CSLL. Also realized I should justify why it needs to be changed lol: 

Right now if you were to cause an error message to be thrown, skip back, and then attempt to do any operation the visualization would freeze. It's really just because the label itself is associated with the ID defined in setup (because of this.resetIndex), but then we change it again here which causes the animation manager to lose track of the old label and crash.